frontend/src/app/main/repo.cljs

```diff
@@ -93,12 +93,12 @@
                     (= query-params :all)  :get
                     (str/starts-with? nid "get-") :get
                     :else :post)
-
         request   {:method method
                    :uri (u/join cf/public-uri "api/rpc/command/" nid)
                    :credentials "include"
                    :headers {"accept" "application/transit+json,text/event-stream,*/*"
-                             "x-external-session-id" (cf/external-session-id)}
+                             "x-external-session-id" (cf/external-session-id)
+                             "x-event-origin" (:app.main.data.events/origin (meta params))}
```

Comment:
Could we avoid using `:app.main.data.events/origin` ?

Reply:
yes, requiring an alias:
```
[app.main.data.events :as-alias ev]
```
on requires

Reply:
that will resultt on `::ev/origun`